.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
I am running 2.5.3 based on the mod intermission guidelines... were we supposed to install 2.4.1 for this project?
https://backend.turing.io/module3/misc/ruby_and_rails_versions

Reply:
Yeah, that's what I had at first. But on this project readme, it states Ruby 2.4.1 so I went with that.